### PR TITLE
[ADD] rename_property

### PR DESCRIPTION
**Pull request by hbrunn**
this is used in https://github.com/OCA/OpenUpgrade/pull/576

### CODE REVIEW

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -1320,3 +1321,22 @@ def savepoint(cr):
             cr.execute('RELEASE SAVEPOINT "%s"' % name)
         except:
             cr.execute('ROLLBACK TO SAVEPOINT "%s"' % name)
+
+
+def rename_property(cr, model, old_name, new_name):
+    """Rename property old_name owned by model to new_name. This should happen
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: You can add the parameter documentation in docstring format with `:param`

**Comment by pedrobaeza** [comment]
I think this should be part of `rename_columns` method, to be transparent for the user if the column is a property or not.

**Comment by hbrunn** [comment]
this would involve a lot of auto-detection-foo that I would not want a low level library to do. do you insist on this or can you live with the extra function if there's no overwhelming support for your suggestion?

**Comment by pedrobaeza** [comment]
OK, if no one supports me, then you can continue with this isolated method.

**Comment by StefanRijnhart** [comment]
:+1: I get @pedrobaeza's suggestion, but rename_columns is a very simplistic sql operation (it is not a proper rename_*fields*).

**Pull request merged**